### PR TITLE
General month/year count

### LINKED ISSUE #2144: Incorrect tooltip on the supporter length slider on Polish language
https://puu.sh/yRBUs/7bad5d6b13.webm

On USD 36 it's `1 year and 4 months`, but later it changes into  just `1 year`

### PR DESCRIPTION
**Pull request by nanaya**
Cleanups. Doesn't \<verb\> #2144 because I don't speak pl ┐(° ヮ° )┌

### CODE REVIEW

**Comment by Aergwyn** [comment]
Sorry I actually chuckled as you cycled through words. :3c

**Comment by nanaya** [comment]
You didn't see anything.

File: resources/lang/pl/common.php [hunk 1]
```diff
@@ -43,6 +43,8 @@
 
     'count' => [
         'item' => ':count sztuka|:count sztuki|:count sztuk',
+        'months' => ':count miesiąc|:count miesięcy',
```
Comment (LiquidPL) [inline resources/lang/pl/common.php]: `'months' => ':count miesiąc|:count miesiące|:count miesięcy'` should be fine.

**Comment by notbakaneko** [comment]
The zero variations were being used by `SupporterTag::getDurationText()`
Now the emails are going to have untranslated keys (☞ﾟ∀ﾟ)☞

**Pull request merged**